hoshino/res.py

```diff
@@ -1,6 +1,7 @@
 import os
+import asyncio
 from PIL import Image
-import requests
+import httpx as requests
```

Comment:
No! This is still sync function. You should use async & await to open a RemoteResImg

Reply:
Replace all sync requests with await/async

Reply:
Sure, we should not block the bot when polling weibo. Remember that we only have 1 thread.